commentsetting.h

```diff
@@ -41,10 +42,11 @@ class CommentSetting {
     static bool appendComment;
     static std::vector<CommentSetting> comments;
 
-    explicit CommentSetting(const QString shortcut, const QString text = "", const QColor color = QColor(255, 255, 0, 255), const float nodeRadius = 1.5);
+    explicit CommentSetting(QString  shortcut, QString  text = "", QColor  color = QColor(255, 255, 0, 255), const float nodeRadius = 1.5) :
+        shortcut(std::move(shortcut)), text(std::move(text)), color(std::move(color)), nodeRadius(nodeRadius) {}
```

Comment:
i would like to know the rationale to put the ctor in the header, it’s fine, but i want to know why

Reply:
Here it just suggested to move the definition to the header, without any explanation and here it doesn't display which clang-tidy option is responsible for this suggestion.